instances/widgets.treasury-factory.near/widget/components/TokenAmount.jsx

```diff
@@ -7,12 +7,20 @@ const amountWithDecimals = props.amountWithDecimals ?? 0;
 const amountWithoutDecimals = props.amountWithoutDecimals;
 
 const isNEAR = address === "" || address.toLowerCase() === "near";
+const isWrapNear = address === "wrap.near";
 
 let ftMetadata = {
   symbol: "NEAR",
   decimals: 24,
 };
-if (!isNEAR) {
+if (isWrapNear) {
+  ftMetadata = {
```

Comment:
The metadata is also available in the `ft_metadata` view method of the contract. Isn't it better to look it up from the contract instead of hardcoding it here?